package.nls.json

```diff
@@ -6,14 +6,16 @@
     "containerApps.enableOutputTimestamps": "Prepends each line displayed in the output channel with a timestamp.",
     "containerApps.browse": "Browse",
     "containerApps.createContainerApp": "Create Container App...",
-    "containerApps.deployImage": "Update Container App Image......",
+    "containerApps.editContainerApp": "Edit Container App (Advanced)",
+    "containerApps.deployImage": "Update Container App Image...",
```

Comment:
I'm wondering if we should call this: "Update Container..." or "Edit Container..." after our conversation offline. The image is just a part of the overall container, after all.

It's annoying and confusing, but Container =/= Container App.

Reply:
😵‍💫